src/derivatives/atlas.md

```diff
@@ -8,45 +8,43 @@ segmentations, and other knowledge annotations such as landmarks defined with
-     a [template](../common-principles.md).
+     a [template](../common-principles.md).  MUST NOT be used alongside `atlas-<label>`
+-    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to specify derivatives defining an [atlas](../common-principles.md).  MUST NOT be used alongside `template-<label>`
 -    [`space-<label>`](../glossary.md#space-entities) is REQUIRED to disambiguate derivatives defined with
      respect to different [coordinate systems](../appendices/coordinate-systems.md), following the general
-     BIDS-Derivatives specifications.
+     BIDS-Derivatives specifications.  It is also REQUIRED when [`tpl-<label>`](../glossary.md#template-entities) is used,
+     and the template is NOT serving as the authoritative definition of a space.
 -    [`cohort-<label>`](../glossary.md#cohort-entities) is REQUIRED to disambiguate derivatives defined with
-     respect to different cohort instances of a single [space (coordinate system)](../appendices/coordinate-systems.md).
+     respect to different cohort instances.
      Please note that [`cohort-<label>`](../glossary.md#cohort-entities) MUST NOT be used if neither
-     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities) are used.
--    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to encode files pertaining
-     or derived from the atlas identified by the entity's label.
+     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities)
+     nor [`atlas-<labl>`](../glossary.md#atlas-entities) are used.
 -    [`seg-<label>`](../glossary.md#segmentation-entities) is REQUIRED when a single atlas has several different
      realizations (for instance, segmentations and parcellations created with different criteria) that
      need disambiguation.
 -    [`scale-<label>`](../glossary.md#scale-entities) is REQUIRED to disambiguate different atlas 'scales',
      when the atlas has more than one 'brain unit' resolutions, typically relating to the area covered
      by regions.
 
-The general filename pattern for subject derivatives with templates and atlases in their provenance
+The general filename pattern for subject-level derivatives with templates and atlases 
 follows the general BIDS-Derivatives pattern:
 
 ```Text
 <pipeline_name>/
     sub-<label>/
         <datatype>/
-            <source_entities>[_space-<space>][_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
+            <source_entities>[_atlas-<label>][_space-<space>][_cohort-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
+            <source_entities>[_tpl-<label>][_space-<space>][_cohort-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```

Comment:
This creates a lot of ambiguity, why would you need both tpl- and space-?

In BIDS-Derivatives, space can take values as `space-anat`, `space-native`, etc. indicating that the authoritative coordinate system (space) is the same subject's scanner coordinates. What is `sub-01_tpl-A_space-B_suffix` trying to say? If `sub-01_space-B_suffix` means that you moved `suffix` from `sub-01` into `space-B`, then, I'm finding it hard to understand what `sub-01_tpl-A_space-B_suffix` means.

The first line reorders atlas and space [+cohort], but it feels more natural to me to define the space first and second the atlas.

```suggestion
           <source_entities>[_space-<space>][_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```